src/styles/blocks/star.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    border-radius: 0.5px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  color: #060b35;
```

Comment:
The color value should be defined using an SCSS variable. Ensure that all main values are stored in a separate variables file as per the task requirements.